script/generate_dispatch.py

```diff
@@ -0,0 +1,387 @@
+									extension_commands[(extension_name, n_required['@feature'])] += [command['@name']]
+						elif 'command' in n_required:
+							commands = [n_required['command']]
+							for command in commands:
+								if type(command) is list:
+									for n_command in command:
+										if n_command['@name'] in device_commands:
+											extension_commands[(extension_name, 'VK_VERSION_1_0')] += [n_command['@name']]
+								elif command['@name'] in device_commands:
+									extension_commands[(extension_name, 'VK_VERSION_1_0')] += [command['@name']]
+				elif 'command' in required:
+					commands = [required['command']]
+					for command in commands:
+						if type(command) is list:
+							for n_command in command:
+								if n_command['@name'] in device_commands:
+									extension_commands[(extension_name, 'VK_VERSION_1_0')] += [n_command['@name']]
+						elif command['@name'] in device_commands:
+							extension_commands[(extension_name, 'VK_VERSION_1_0')] += [command['@name']]
+
+# Build header
+
+# License
+header = '/* \n'
+header += ' * Copyright © 2021 Cody Goodson (contact@vibimanx.com)\n'
+header += ' * \n'
+header += ' * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated\n'
+header += ' * documentation files (the “Software”), to deal in the Software without restriction, including without\n'
+header += ' * limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies\n'
+header += ' * of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:\n'
+header += ' * \n'
+header += ' * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.\n'
+header += ' * \n'
+header += ' * THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT\n'
+header += ' * LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.\n'
+header += ' * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,\n'
+header += ' * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.\n'
+header += ' * \n'
+header += ' */\n\n'
+
+# Info
+header += '// This file is a part of VkBootstrap\n'
+header += '// https://github.com/charles-lunarg/vk-bootstrap\n\n'
+
+# Content
+header += '\n#pragma once\n\n#include <vulkan/vulkan.h>\n\n'
+header += 'namespace vkb {\n\n'
+header += 'struct DispatchTable {\n'
+header += '\tDispatchTable() = default;\n'
+header += '\tDispatchTable(VkDevice const& device, PFN_vkGetDeviceProcAddr const& procAddr) : device(device) {\n'
+
+proxy_definition = ''
+pfn_declaration = ''
+pfn_loading = ''
+
+for level in core_commands:
+	pfn_declaration += '#ifdef ' + level + '\n';
```

Comment:
Consider using 'string interpolation' instead of adding everything together with +'s. This makes the code a lot more legible and fun to write.
https://www.programiz.com/python-programming/string-interpolation